bluepill/BPVersion.h

```diff
@@ -0,0 +1 @@
+#define BP_VERSION "v4.0.0-3-g75181da"
```

Comment:
can we only have one place to specify the version? We have it here & in VERSION

Reply:
Oops, good catch... this file was checked in by mistake. There should be only _one_ `BPVersion.h` file that is generated in the `bp` directory. I also added an API to get the version instead of having every place read `BP_VERSION` directly. I'll remove this file.